datamade/civic-scraper
# factor out Legistar scraper similarities

## Overview
This PR factors out commonalities between the scrapers for Canton, GA; Smyrna City, GA; and Wellington, GA Legistar instances.

## Testing
* Run `docker-compose run --rm scraper python -m civic_scraper.platforms.legistar.{scraper_name}` where scraper_name is `ga_canton`, `ga_smyrna`, and `ga_wellington`.
* Make sure the scraper runs successfully

Handles #5
